feature-styles/PS_ProtectedSites.se

```diff
@@ -8,7 +8,7 @@
   <se:FeatureTypeName>ps:ProtectedSite</se:FeatureTypeName>
   <se:Rule>
     <se:Description>
-      <se:Title>protected sites: polygon</se:Title>
+      <se:Title>Protected sites: polygon</se:Title>
```

Comment:
For other FeatureTypes with different geometries the colon between Type and geometry has been removed - maybe align.